src/test/java/sirius/web/security/ProfileTest.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.security
+
+import sirius.kernel.BaseSpecification
+import sirius.kernel.Sirius
+
+class ProfileTest extends BaseSpecification {
+
+    def "profile cascading is invalid when cascading to lower priority"() {
+        when:
+        Profile.compile(Sirius.getSettings().getExtension("security.profiles",
+                                                          "test-cascade-to-target-with-lower-priority"))
+               .validate()
+        then:
+        def e = thrown(IllegalStateException)
+        e.getMessage() == "Profile 'test-cascade-to-target-with-lower-priority' refers to a profile which is applied " +
```

Comment:
can be exported into ```createErrorMessage(profile1,profile2)```to reduce test maintenace cost when editing this error message